client/state/posts/utils.js

```diff
@@ -29,7 +35,7 @@ export function getNormalizedPostsQuery( query ) {
  */
 export function getSerializedPostsQuery( query = {}, siteId ) {
 	const normalizedQuery = getNormalizedPostsQuery( query );
-	const serializedQuery = JSON.stringify( normalizedQuery ).toLocaleLowerCase();
+	const serializedQuery = JSON.stringify( normalizedQuery );
```

Comment:
>Why the change here?
 

I added an extended description to 988ea7d26d6c4bf62efe894ba9719182f6656c87 which explains why `toLowerCase` is a nice convenience for deduping, but in this case prevents accurately deserializing the key.

Related: `QueryManagerKey` [does not lowercase the key](https://github.com/Automattic/wp-calypso/blob/8238571290db36b15e986fd04c83c3311708d8b0/client/lib/query-manager/key.js#L20), and hasn't been a problem.